library/accessory_workflows/build_cellranger_reference/BuildCellRangerReferenceBundle.wdl

```diff
@@ -0,0 +1,80 @@
+task BuildCellRangerReference {
+  String ref_fasta
+  String gtf_file
+  String ref_name
+
+  String docker = "quay.io/humancellatlas/secondary-analysis-cellranger"
+  String memory = "416 GB"
```

Comment:
Do we definitely need a machine this big?

Reply:
https://cloud.google.com/compute/docs/machine-types is a good info to know! So let me use 60 GB for a try. When it comes to CPUs, I used `getconf _NPROCESSORS_ONLN` which will fetch the number of available CPU cores in the runtime.

Reply:
Oh, I see, the original settings result in getting n1-highmem-64. However, since the documentation says we don't need so much memory, it might make sense to use n1-highcpu-64 instead, which we should get if we ask for 64 cpus and ~50 GB of memory.

Reply:
we asked for 50G when we built Star reference for  SS2. it should be adequate amount. The lower MEM might work as well.